src/app/zap-templates/zcl/data-model/chip/smoke-co-alarm-cluster.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2008,2020 Silicon Labs.
```

Comment:
I _really_ doubt this is the right copyright date.

Reply:
Fixed in the new commit.